client/executor/src/integration_tests/mod.rs

```diff
@@ -64,6 +67,15 @@ macro_rules! test_wasm_execution {
 			}
 		}
 	};
+
+	(compiled_only $method_name:ident) => {
+		paste::item! {
+			#[test]
+			fn [<$method_name _compiled>]() {
+				$method_name(WasmExecutionMethod::Compiled);
+			}
+		}
+	};
```

Comment:
I don't see this used anywhere. Why is this added here?

Reply:
Good catch. It was used in previous iteration but not anymore